.github/pull_request_template.md

```diff
@@ -4,7 +4,7 @@ Thank you for your Pull Request! We have developed this task checklist from the
 Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.
 
 - [ ] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
-- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
+- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
```

Comment:
There's `styler::style_package()` which styles all files in a package. Maybe we should suggest to use that function instead? On the other hand, maybe we just want to keep it to the newly added and/or changed files.

Reply:
yeh i would leave it or it would confuse new contributors if they ran it and had other files changing that they didn't touch with any edits

Reply:
we can always run that one as part of a release before merging devel to main